packages/contracts-bedrock/scripts/deploy/DeployImplementations.s.sol

```diff
@@ -360,25 +358,25 @@ contract DeployImplementationsOutput is BaseDeployIO {
     function assertValidL1CrossDomainMessengerImpl(DeployImplementationsInput) internal view {
         IL1CrossDomainMessenger messenger = l1CrossDomainMessengerImpl();
 
-        DeployUtils.assertInitialized({ _contractAddress: address(messenger), _slot: 0, _offset: 20 });
+        DeployUtils.assertInitialized({ _contractAddress: address(messenger), _isProxy: false, _slot: 0, _offset: 20 });
 
-        require(address(messenger.OTHER_MESSENGER()) == Predeploys.L2_CROSS_DOMAIN_MESSENGER, "L1xDM-10");
-        require(address(messenger.otherMessenger()) == Predeploys.L2_CROSS_DOMAIN_MESSENGER, "L1xDM-20");
+        require(address(messenger.OTHER_MESSENGER()) == address(0), "L1xDM-10");
+        require(address(messenger.otherMessenger()) == address(0), "L1xDM-20");
         require(address(messenger.PORTAL()) == address(0), "L1xDM-30");
         require(address(messenger.portal()) == address(0), "L1xDM-40");
         require(address(messenger.superchainConfig()) == address(0), "L1xDM-50");
 
         bytes32 xdmSenderSlot = vm.load(address(messenger), bytes32(uint256(204)));
-        require(address(uint160(uint256(xdmSenderSlot))) == Constants.DEFAULT_L2_SENDER, "L1xDM-60");
+        require(address(uint160(uint256(xdmSenderSlot))) == address(0), "L1xDM-60");
     }
 
     function assertValidL1ERC721BridgeImpl(DeployImplementationsInput) internal view {
         IL1ERC721Bridge bridge = l1ERC721BridgeImpl();
 
-        DeployUtils.assertInitialized({ _contractAddress: address(bridge), _slot: 0, _offset: 0 });
+        DeployUtils.assertInitialized({ _contractAddress: address(bridge), _isProxy: false, _slot: 0, _offset: 0 });
```

Comment:
ForgeArtifacts.sol has a [getInitializedSlot](https://github.com/ethereum-optimism/optimism/blob/upgrade-against-registry-2/packages/contracts-bedrock/scripts/libraries/ForgeArtifacts.sol#L144) method you can use for the slot an offset. Currently these feel like magic number.

Reply:
implemented 🫡

Reply:
This caused issues with op-deployer, so we agreed this is not necessary. Resolving.